.ci/scripts/send-pr.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -uexo pipefail
+
+readonly APM_AGENT=${1}
+readonly APM_AGENT_SPECS_DIR=${2}
+readonly APM_AGENT_REPO_NAME="apm-agent-${APM_AGENT}"
+
+mkdir -p .ci/git
+git clone https://github.com/elastic/${APM_AGENT_REPO_NAME} .ci/git/${APM_AGENT_REPO_NAME}
+
+mkdir -p .ci/git/${APM_AGENT_REPO_NAME}/${APM_AGENT_SPECS_DIR}
+cp tests/agents/gherkin-specs/*.feature .ci/git/${APM_AGENT_REPO_NAME}/${APM_AGENT_SPECS_DIR}
+
+cd .ci/git/${APM_AGENT_REPO_NAME}
+git checkout -b update-feature-files-$(date "+%Y%m%d%H%M%S")
+echo "Copying feature files to the ${APM_AGENT_REPO_NAME} repo"
+git status
```

Comment:
Is this necessary once this goes into production?

Reply:
Possibly not, evolving this script step-by-step for fast feedback

Reply:
Sounds good. I'll just leave this thread open as a reminder to remove it before it gets merged. Thanks!

Reply:
Removed!